nixos/modules/services/misc/sdrplay.nix

```diff
@@ -0,0 +1,15 @@
+{ config, lib, pkgs, sdrplay, ... }:
+with lib;
+{
+  options.services.sdrplay_api = {
```

Comment:
```suggestion
  options.services.sdrplayApi = {
```

Reply:
I changed the name according to your suggestion.